backend/src/packages/chaiNNer_ncnn/ncnn/processing/upscale_image.py

```diff
@@ -189,6 +204,7 @@ def upscale_image_node(
     img: np.ndarray,
     model: NcnnModelWrapper,
     tile_size: TileSize,
+    custom_tile_size: int,
```

Comment:
I would prefer for this to be an optional int, so that None can be passed instead of 0 for those other places that call the node, just to be a bit clearer